packages/mobile/src/screens/edit-track-screen/types.ts

```diff
@@ -13,6 +13,8 @@ export type FormValues = Omit<TrackMetadataForUpload, 'bpm'> & {
   trackArtwork?: string
   bpm?: string
   isUpload?: boolean
+  is_private?: boolean
```

Comment:
should this be `is_unlisted`? or can this thing be a `CollectionMetadata` also?
Wondering since the type is based on `Omit<TrackMetadataForUpload, 'bpm'>`

Reply:
yeah this is a hack-alert to make FormValues quickly work with collections also, will add it to that clean up ticket. great spots dude.